plugins/flowdock/lib/samson_flowdock/samson_plugin.rb

```diff
@@ -20,22 +21,47 @@ def flowdock_tokens
     def no_flowdock_token?(flowdock_attrs)
       flowdock_attrs['token'].blank?
     end
+
+    def enabled_flows_names
+      flowdock_flows.enabled.map(&:name)
+    end
+  end
+end
+
+Samson::Hooks.callback :deploy_defined do
+  Deploy.class_eval do
```

Comment:
When using ```class Deploy; ....; end``` syntax it always creates a class, or redefines it if it exists, failing silently if you did a typo.

When we use ```class_eval``` it assumes a class exists, and if it doesn't this will cause ```const_missing``` to be called on the parent module which usually throws an error.
Rails overrides ```const_missing``` and autoloads the class, throwing an error only if it cannot find it in the load path.